CHANGELOG.md

```diff
@@ -129,6 +129,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Removed the button to remove an agent for a group in the agents' table when it is the default group [#3804](https://github.com/wazuh/wazuh-kibana-app/pull/3804)
 - Fixed internal user no longer needs permission to make x-pack detection request [#3831](https://github.com/wazuh/wazuh-kibana-app/pull/3831)
 - Fixed agents details card style [#3845](https://github.com/wazuh/wazuh-kibana-app/pull/3845)
+- Disable configuration checksum sorting in groups management table [#3857](https://github.com/wazuh/wazuh-kibana-app/pull/3857)
```

Comment:
**thougth**: Probably, the change in the changelog should be moved to another version/revision depending on the version this will be released.